web/js/scripts.js

```diff
@@ -63,24 +63,33 @@ var grouphub = (function ($) {
         });
     };
 
-    var lowerGroupCount = function (groupId) {
-        var $counts = $('.group-' + groupId).find('.count');
+    var incrementGroupMemberCount = function (groupId) {
+        var newMemberCount = getGroupMemberCount(groupId) + 1;
 
-        $counts.each(function () {
-            var $this = $(this);
+        updateGroupMemberCounters(groupId, newMemberCount);
+    };
 
-            $this.html(parseInt($this.text(), 10) - 1);
-        });
+    var decrementGroupMemberCount = function (groupId) {
+        var newMemberCount = getGroupMemberCount(groupId) - 1;
+
+        updateGroupMemberCounters(groupId, newMemberCount);
     };
 
-    var raiseGroupCount = function (groupId) {
-        var $counts = $('.group-' + groupId).find('.count');
+    var getGroupMemberCount = function (groupId) {
+        var $groupMemberCounts = $('.group-' + groupId).find('.count');
 
-        $counts.each(function () {
-            var $this = $(this);
+        return parseInt($groupMemberCounts.first().text(), 10);
+    };
 
-            $this.html(parseInt($this.text(), 10) + 1);
+    var updateGroupMemberCounters = function (groupId, newCount) {
+        var $groupMemberCounts = $('.group-' + groupId).find('.count');
+
+        $groupMemberCounts.each(function () {
+            $(this).html(newCount);
         });
+
+        var $editGroupMemberCount = $('.edit_group .count');
+        $editGroupMemberCount.text('(' + newCount + ')');
     };
```

Comment:
It's a shame state is managed through actual DOM objects (you would rather maintain a data model in JS), but splitting the logic in separate functions mitigates this pain a bit. 

Alternatively, you could maintain a `data-group-members-count` on an element containing a group and its members. This decouples the data model a bit from the DOM, but has the trade-off that one would have to read and write to this variable to keep the count in sync. For now, I'd leave it be as most of the JS is not written in this style.